lighthouse-core/runner.js

```diff
@@ -107,6 +107,12 @@ class Runner {
       // Entering: conclusion of the lighthouse result object
       const lighthouseVersion = require('../package.json').version;
 
+      // Use version from gathering stage. In case accessibility category doesn't run,
+      // fallback to package json so the report will always have some version.
+      const axeVersion = artifacts.Accessibility ?
```

Comment:
The failure-failure mode is when `artifacts.Accessibility` exists but is an `Error` object because something threw.

Reply:
let's just not show in runtime settings if we didn't find a version in the artifact